### PR TITLE
Adds veBAL APR

### PR DESCRIPTION
**Pull request by garethfuller**
# Description

Adds logic for calculating and displaying veBAL APRs.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

- [ ] Test APRs for the BAL 80/20 pool on mainnet include veBAL APR
- [ ] Test Lido (wstETH) pool, I had to do a minor cleanup around price fetching there.
- [ ] Test pool creation, including entering manual prices for unknown tokens.

## Visual context

Screenshot 2022-05-27 at 14 24 12

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/composables/queries/useTokenPricesQuery.ts [hunk 1]
```diff
@@ -27,37 +27,20 @@ const PER_PAGE = 1000;
...
-    const wstEthAddress = configService.network.addresses.wstETH;
-    if (prices[stEthAddress]) {
-      const stETHPrice = prices[stEthAddress][currency.value] || 0;
-      prices[wstEthAddress] = {
-        [currency.value]: TOKENS.Prices.ExchangeRates.wstETH.stETH * stETHPrice
-      };
-    }
-
-    return prices;
-  }
```
Comment (garethfuller) [inline src/composables/queries/useTokenPricesQuery.ts:51]: wstETH is now supported in Coingecko! Yay!

https://api.coingecko.com/api/v3/simple/token_price/ethereum?contract_addresses=0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0&vs_currencies=usd

File: src/locales/default.json [hunk 2]
```diff
@@ -989,6 +989,11 @@
         "withdraw": {
             "priceImpact": "Price impact from removing liquidity results when the value of each token removed is not proportional to the weights of the pool. Removing non-proportional amounts causes the internal prices of the pool to change, as if you were swapping tokens. The higher the price impact, the worse price you’ll get for exiting your position.",
             "total": "The total value in {0} you’ll be withdrawing from this pool."
+        },
+        "veBalApr": {
+            "title": "Locking / veBAL APR",
```
Comment (markusbkoch) [inline src/locales/default.json:994]: @garethfuller @pkattera  We might want to word this as "Max locking/veBAL APR" or "Max veBAL APR" for short

Comment (pkattera) [reply src/locales/default.json:994]: Happy with either of those suggestions.

**Comment by timjrobinson** [comment]
It works for me in prod. There were some errors testing in Kovan which is causing a red "Failed to load pools banner":

- bbaUSD address was missing
- Pool warnings was missing
- Fee distributor address is missing

I fixed the first two but couldn't find the address for the fee distributor on Kovan. Do we have one?

File: src/components/modals/UnknownTokenPrice/UnknownTokenPriceModal.vue [hunk 3]
```diff
@@ -36,13 +38,18 @@ const { t } = useI18n();
...
+const unknownTokenPrices = computed(
+  (): TokenPrices => {
+    const _unknownTokenPrices = {};
+    for (const token of props.unknownTokens) {
+      _unknownTokenPrices[token] = {
+        [FiatCurrency.usd]:
+          injectedPrices.value[FiatCurrency.usd][token] || null
+      };
+    }
+    return _unknownTokenPrices;
```
[the remark this answers is not in the document]
Comment (timjrobinson) [reply src/components/modals/UnknownTokenPrice/UnknownTokenPriceModal.vue:50]: This breaks the model binding, as it binds a number/string to injectedPrices in the template.

File: src/locales/default.json (hunk 2 above)
Comment (garethfuller) [reply src/locales/default.json]: Updated

**Comment by garethfuller** [comment]
Thanks @timjrobinson, I've addressed both your comments.

**Pull request merged**